shub/deploy.py

```diff
@@ -56,34 +60,11 @@
 SHORT_HELP = "Deploy Scrapy project to Scrapy Cloud"
 
 
-_SETUP_PY_TEMPLATE = """\
-# Automatically created by: shub deploy
-
-from setuptools import setup, find_packages
-
-setup(
-    name         = 'project',
-    version      = '1.0',
-    packages     = find_packages(),
-    entry_points = {'scrapy': ['settings = %(settings)s']},
-)
-"""
-
-
-def list_targets(ctx, param, value):
-    if not value:
-        return
-    conf = load_shub_config()
-    for name in conf.projects:
-        click.echo(name)
-    ctx.exit()
-
-
 @click.command(help=HELP, short_help=SHORT_HELP)
 @click.argument("target", required=False, default="default")
 @click.option("-l", "--list-targets", help="list available targets",
               is_flag=True, is_eager=True, expose_value=False,
-              callback=list_targets)
+              callback=list_targets_callback)
```

Comment:
Maybe `List available project names defined in your config`?
I used a part of `target` [definition](https://shub.readthedocs.io/en/stable/deploying.html#deploying-projects) from existing shub docs.